app/src/main/java/com/ataulm/whatsnext/search/SearchViewModel.kt

```diff
@@ -24,6 +28,12 @@ internal class SearchViewModel(private val service: WhatsNextService) : ViewMode
     fun onClick(filmSummary: FilmSummary) {
         _navigationEvents.value = Event(filmSummary)
     }
+
+    fun pagedPopularFilms(): Flow<PagingData<FilmSummary>> {
+        val popularFilmsPagingSource = service.popularFilmsThisWeek()
+        val pager = Pager(config = PagingConfig(pageSize = 9)) { popularFilmsPagingSource }
```

Comment:
9 is not a legit number, needs revisiting after I figure out what the UI will look like